tests/app/celery/test_tasks.py

```diff
@@ -98,8 +98,9 @@ def email_job_with_placeholders(notify_db, notify_db_session, sample_email_templ
     return create_job(template=sample_email_template_with_placeholders)
 
 
+@pytest.mark.usefixtures("notify_db_session")
 class TestBatchSaving:
-    def test_should_save_smss(self, notify_db_session, sample_template_with_placeholders, mocker):
```

Comment:
heh, I like this fixture removal - I find it odd how the pattern is to include it in so many of our unit tests when it is rarely used

Reply:
Yeah there might be a better pattern to teardown the database tables and session than specifying this. Using test classes though allowed me to move this at line 101 instead, so it will be used for all tests with class. 👍